atomic_physics/ions/ba133.py

```diff
@@ -0,0 +1,132 @@
+""" 133Ba+
+
+ToDo: We need a reference for the Einstein A values
```

Comment:
can you open an issue for this and remove the TODO so it doesn't get forgotten about plz?